packages/vats/src/vat-bank.js

```diff
@@ -186,12 +186,13 @@ const prepareBankChannelHandler = zone =>
     /** @param {MapStore<string, MapStore<string, BalanceUpdater>>} denomToAddressUpdater */
     denomToAddressUpdater => ({ denomToAddressUpdater }),
     {
+      /** @param {BridgeMessage} obj */
       async fromBridge(obj) {
-        switch (obj && obj.type) {
+        switch (obj?.type) {
           case 'VBANK_BALANCE_UPDATE': {
             const { denomToAddressUpdater } = this.state;
             for (const update of obj.updated) {
-              const { address, denom, amount: value, nonce } = update;
```

Comment:
@michaelfig does f80a82f7126d0414928c23f0a3fffea73e0369f0 look right to you?

Reply:
Yes, that looks correct.  The old code worked because our balance updates are queued to vat-bank in the order they were sent, so the nonce doesn't have any effect.  The only thing the nonce guards against is updates arriving at vat-bank out-of-order from when they are sent (which doesn't happen in SwingSet's current implementation of message delivery).

Reply:
Actually there is a strong guarantee in the SwingSet execution model that 2 messages to the same object will be delivered in the same order as they were sent (if sent from the same party, in this case cosmic-swingset through the bridge vat).

As such I believe we should be allowed to rely on this guarantee, which in this case make the `nonce` redundant, but also tells us that by missing it, vat bank would not have been affected by any changes to the execution model that kept satisfying the message delivery guarantees we've committed to.